src/components/Navigation/Ui/ScoreRenderer.js

```diff
@@ -71,28 +72,56 @@ class ScoreRenderer extends Component {
 
           <h2>Licensed</h2>
           {this.renderScores(licensedScore, licensedToolScore)}
+          {this.renderScoreInfo()}
         </div>
       )
     } else {
-      return <div className="ScoreRenderer">{this.renderScores(get(domain, 'score'), get(domain, 'toolScore'))}</div>
+      return (
+        <div className="ScoreRenderer">
+          {this.renderScores(get(domain, 'score'), get(domain, 'toolScore'))} {this.renderScoreInfo()}
+        </div>
+      )
     }
   }
 
   renderScores(effective, tools) {
     return (
       <div className="ScoreRenderer__domain">
         <div className="ScoreRenderer__domain__section">
-          <h2>{`Effective: ${isNumber(effective.total) ? effective.total : effective}`}</h2>
+          <h2>
+            {`Effective: ${isNumber(effective.total) ? effective.total : effective}`}
+            {this.renderUnit()}
+          </h2>
           {this.renderScore(effective)}
         </div>
         <div className="ScoreRenderer__domain__section">
-          <h2>{`Tools: ${isNumber(tools.total) ? tools.total : tools}`}</h2>
+          <h2>
+            {`Tools: ${isNumber(tools.total) ? tools.total : tools}`}
+            {this.renderUnit()}
+          </h2>
           {this.renderScore(tools)}
         </div>
       </div>
     )
   }
 
+  renderUnit = () => (
+    <span>
+      / 100 <span className="unit">points</span>
```

Comment:
Let's drop "points" everywhere and make sure the spacing around the slash is even.